### PR TITLE
Renamed the Exposure field to Exposure Compensation in sky volume overrides (case 1392530).

### PR DESCRIPTION
**Pull request by anisunity**
https://fogbugz.unity3d.com/f/cases/1392530/
Pretty straight forward rename

Testing status:
Made sure it appears properly in the volume components

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Sky/SkySettingsEditor.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ protected enum SkySettingsUIElement
         }
 
         GUIContent m_SkyIntensityModeLabel = new UnityEngine.GUIContent("Intensity Mode");
+        GUIContent m_ExposureCompensationLabel = new GUIContent("Exposure Compensaiton", "Sets the exposure compensation of the sky in EV.");
```
Comment (remi-chapelain) [inline com.unity.render-pipelines.high-definition/Editor/Sky/SkySettingsEditor.cs:28]: Quick typo here, "compasaition"

**Review by remi-chapelain — CHANGES_REQUESTED** [review]
Only quick typo otherwise, looks good to me.

**Review by remi-chapelain — APPROVED** [review]
LGTM ✔️

File: com.unity.render-pipelines.high-definition/Editor/Sky/SkySettingsEditor.cs (hunk 1 above)
Comment (anisunity) [reply com.unity.render-pipelines.high-definition/Editor/Sky/SkySettingsEditor.cs]: woopsie

**Pull request merged**